signer/src/config/default.toml

```diff
@@ -42,17 +42,15 @@ max_retry_attempts = 5
 ping_interval = 60
 subscribe_interval = 10
 
-# !! ==============================================================================
```

Comment:
Realized that this is for the Hiro API, but couldn't find that it's used anywhere so I removed it.

Reply:
🙏🏽